target/linux/ar71xx/files/arch/mips/ath79/Makefile

```diff
@@ -84,6 +84,7 @@ obj-$(CONFIG_ATH79_MACH_DLAN_PRO_500_WP)	+= mach-dlan-pro-500-wp.o
 obj-$(CONFIG_ATH79_MACH_DR344)			+= mach-dr344.o
 obj-$(CONFIG_ATH79_MACH_DR531)			+= mach-dr531.o
 obj-$(CONFIG_ATH79_MACH_DRAGINO2)		+= mach-dragino2.o
+obj-$(CONFIG_ATH79_MACH_DOMYWIFI_DW33D)         += mach-domywifi-dw33d.o
 obj-$(CONFIG_ATH79_MACH_EAP300V2)		+= mach-eap300v2.o
```

Comment:
please use tabs here and not spaces. also DO is before DR, please move the line up by one